README.md

```diff
@@ -15,6 +15,7 @@ To contribute:
 
 | Name                                                                                                                                                       | Location                      | Application Period    | Notes                                                                                          |
 | ---------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------- | --------------------- | ---------------------------------------------------------------------------------------------- |
+| [Mastercard](https://careers.mastercard.com/us/en/job/R-201217/Software-Engineer-Intern-Summer-2024-Vancouver-Canada)                                                                              | Vancouver                       | Unknown               | Software Engineer, Intern - Vancouver (Summer 2024)                                              |
```

Comment:
The job description states that it is a fulltime role 🤔 . They want you to graduate in Summer 2024 lol, weird

Reply:
My bad, sorry😥.

Reply:
No, don't blame you, it could be either of the following:
- It is an intern role, but they forgot to update the description
- It is a Full-time role but they forgot to edit the title